actors/power/src/types.rs

```diff
@@ -65,3 +66,29 @@ pub struct CurrentTotalPowerReturn {
     pub pledge_collateral: TokenAmount,
     pub quality_adj_power_smoothed: FilterEstimate,
 }
+
+#[derive(Serialize_tuple, Deserialize_tuple)]
+#[serde(transparent)]
+pub struct NetworkRawPowerReturn {
+    #[serde(with = "bigint_ser")]
+    pub raw_byte_power: StoragePower,
+}
+
+impl Cbor for NetworkRawPowerReturn {}
+
+#[derive(Serialize_tuple, Deserialize_tuple)]
```

Comment:
Nit: Add `Debug, Clone, Eq, PartialEq,` to all of these. We often end up wanting it for testing and the compiler will strip from the built artefacts.